pyproject.toml

```diff
@@ -73,7 +72,7 @@ shapely = { version = ">=1.7.0", optional = true } # groups = ["user", "all"]
 pylibjpeg-libjpeg = { version = ">=1.3.2", optional = true } # groups = ["user", "all"]
 
 # AI
-anthropic = { version = ">=0.25.3", optional = true } # groups = ["user", "all"]
+anthropic = { version = "0.25.9", optional = true } # groups = ["user", "all"]
```

Comment:
worth discussing why it's pinned.

Reply:
We'll unpin it soon, just opted to pin so that we don't have suprises